2024-06-04 07:29:09,628 - DFI => Execução iniciada
2024-06-04 07:29:09,669 -     => Main - Execução iniciada
2024-06-04 07:29:09,670 -     => balancasp6000 - Execução iniciada
2024-06-04 07:58:25,305 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 08:01:00,005 -     => balancasp6000 - Execução concluída
2024-06-04 08:01:00,009 -     => Main - Execução concluída
2024-06-04 08:01:00,010 - DFI => Execução encerrada
2024-06-04 09:20:46,952 - DFI => Execução iniciada
2024-06-04 09:20:46,986 -     => Main - Execução iniciada
2024-06-04 09:20:46,988 -     => balancasp6000 - Execução iniciada
2024-06-04 09:21:57,770 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 09:23:14,937 -     => balancasp6000 - Execução concluída
2024-06-04 09:23:14,938 -     => Main - Execução concluída
2024-06-04 09:23:14,938 - DFI => Execução encerrada
2024-06-04 09:27:17,850 - DFI => Execução iniciada
2024-06-04 09:27:17,875 -     => Main - Execução iniciada
2024-06-04 09:27:17,876 -     => balancasp6000 - Execução iniciada
2024-06-04 09:27:45,410 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 09:28:36,021 -     => balancasp6000 - Execução concluída
2024-06-04 09:28:36,021 -     => Main - Execução concluída
2024-06-04 09:28:36,021 - DFI => Execução encerrada
2024-06-04 09:37:46,110 - DFI => Execução iniciada
2024-06-04 09:37:46,134 -     => Main - Execução iniciada
2024-06-04 09:37:46,136 -     => balancasp6000 - Execução iniciada
2024-06-04 09:38:48,141 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 09:39:26,203 -     => balancasp6000 - Execução concluída
2024-06-04 09:39:26,203 -     => Main - Execução concluída
2024-06-04 09:39:26,203 - DFI => Execução encerrada
2024-06-04 09:42:32,919 - DFI => Execução iniciada
2024-06-04 09:42:32,946 -     => Main - Execução iniciada
2024-06-04 09:42:32,947 -     => balancasp6000 - Execução iniciada
2024-06-04 09:45:07,579 - DFI => Execução iniciada
2024-06-04 09:45:07,603 -     => Main - Execução iniciada
2024-06-04 09:45:07,605 -     => balancasp6000 - Execução iniciada
2024-06-04 09:45:50,908 - DFI => Execução iniciada
2024-06-04 09:45:50,937 -     => Main - Execução iniciada
2024-06-04 09:45:50,938 -     => balancasp6000 - Execução iniciada
2024-06-04 09:46:26,994 - DFI => Execução iniciada
2024-06-04 09:46:27,019 -     => Main - Execução iniciada
2024-06-04 09:46:27,020 -     => balancasp6000 - Execução iniciada
2024-06-04 09:50:18,179 - DFI => Execução iniciada
2024-06-04 09:50:18,204 -     => Main - Execução iniciada
2024-06-04 09:50:18,205 -     => balancasp6000 - Execução iniciada
2024-06-04 10:04:20,662 - DFI => Execução iniciada
2024-06-04 10:04:20,701 -     => Main - Execução iniciada
2024-06-04 10:04:20,702 -     => balancasp6000 - Execução iniciada
2024-06-04 10:10:43,492 - DFI => Execução iniciada
2024-06-04 10:10:43,519 -     => Main - Execução iniciada
2024-06-04 10:10:43,520 -     => balancasp6000 - Execução iniciada
2024-06-04 10:11:46,605 - DFI => Execução iniciada
2024-06-04 10:11:46,629 -     => Main - Execução iniciada
2024-06-04 10:11:46,630 -     => balancasp6000 - Execução iniciada
2024-06-04 10:13:36,864 - DFI => Execução iniciada
2024-06-04 10:13:36,889 -     => Main - Execução iniciada
2024-06-04 10:13:36,890 -     => balancasp6000 - Execução iniciada
2024-06-04 10:13:42,470 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:13:47,482 -     => balancasp6000 - Execução concluída
2024-06-04 10:13:47,483 -     => Main - Execução concluída
2024-06-04 10:17:39,798 - DFI => Execução iniciada
2024-06-04 10:17:39,824 -     => Main - Execução iniciada
2024-06-04 10:17:39,825 -     => balancasp6000 - Execução iniciada
2024-06-04 10:18:26,458 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:18:32,953 -     => balancasp6000 - Execução concluída
2024-06-04 10:18:32,953 -     => Main - Execução concluída
2024-06-04 10:21:33,043 -     => Main - Execução iniciada
2024-06-04 10:21:33,046 -     => balancasp6000 - Execução iniciada
2024-06-04 10:22:32,456 - DFI => Execução iniciada
2024-06-04 10:22:32,482 -     => Main - Execução iniciada
2024-06-04 10:22:32,483 -     => balancasp6000 - Execução iniciada
2024-06-04 10:22:52,496 - DFI => Execução iniciada
2024-06-04 10:22:52,521 -     => Main - Execução iniciada
2024-06-04 10:22:52,522 -     => balancasp6000 - Execução iniciada
2024-06-04 10:26:44,969 - DFI => Execução iniciada
2024-06-04 10:26:44,996 -     => Main - Execução iniciada
2024-06-04 10:26:44,997 -     => balancasp6000 - Execução iniciada
2024-06-04 10:26:56,627 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:26:57,278 -     => balancasp6000 - Execução concluída
2024-06-04 10:26:57,278 -     => Main - Execução concluída
2024-06-04 10:29:57,372 -     => Main - Execução iniciada
2024-06-04 10:29:57,373 -     => balancasp6000 - Execução iniciada
2024-06-04 10:30:47,088 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:30:48,580 -     => balancasp6000 - Execução concluída
2024-06-04 10:30:48,580 -     => Main - Execução concluída
2024-06-04 10:33:48,678 -     => Main - Execução iniciada
2024-06-04 10:33:48,680 -     => balancasp6000 - Execução iniciada
2024-06-04 10:34:04,475 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:34:05,257 -     => balancasp6000 - Execução concluída
2024-06-04 10:34:05,258 -     => Main - Execução concluída
2024-06-04 10:37:05,344 -     => Main - Execução iniciada
2024-06-04 10:37:05,345 -     => balancasp6000 - Execução iniciada
2024-06-04 10:37:21,373 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:37:22,265 -     => balancasp6000 - Execução concluída
2024-06-04 10:37:22,265 -     => Main - Execução concluída
2024-06-04 10:39:11,501 - DFI => Execução iniciada
2024-06-04 10:39:11,530 -     => Main - Execução iniciada
2024-06-04 10:39:11,531 -     => balancasp6000 - Execução iniciada
2024-06-04 10:39:12,694 -       => Error:       (X) tools.py.[verifyFile]: O arquivo dfi_balancasp6000_log-2024-06-04.log não foi encontrado no caminho D:\Development\Python\dfi\arquivos-historico\balancasp6000.: 
2024-06-04 10:39:12,766 -     => balancasp6000 - Execução concluída
2024-06-04 10:39:12,767 -     => Main - Execução concluída
2024-06-04 11:02:29,464 - DFI => Execução iniciada
2024-06-04 11:02:29,483 -     => Main - Execução iniciada
2024-06-04 11:02:29,484 -     => commodities - Execução iniciada
2024-06-04 11:02:31,864 -       => Error:       (X) importXlsToDataframe.py.[importXlsSeriesToDataframe]: Importação de arquivos em série Xlsx para Dataframe principal:: unionDataframes() missing 1 required positional argument: 'typeConnect'
2024-06-04 11:04:41,256 - DFI => Execução iniciada
2024-06-04 11:04:41,264 -     => Main - Execução iniciada
2024-06-04 11:04:41,265 -     => commodities - Execução iniciada
2024-06-04 11:05:09,245 - DFI => Execução iniciada
2024-06-04 11:05:09,260 -     => Main - Execução iniciada
2024-06-04 11:05:09,261 -     => commodities - Execução iniciada
2024-06-04 11:06:00,309 - DFI => Execução iniciada
2024-06-04 11:06:00,347 -     => Main - Execução iniciada
2024-06-04 11:06:00,348 -     => commodities - Execução iniciada
2024-06-04 11:06:31,309 - DFI => Execução iniciada
2024-06-04 11:06:31,334 -     => Main - Execução iniciada
2024-06-04 11:06:31,335 -     => commodities - Execução iniciada
2024-06-04 11:07:36,362 - DFI => Execução iniciada
2024-06-04 11:07:36,388 -     => Main - Execução iniciada
2024-06-04 11:07:36,389 -     => commodities - Execução iniciada
2024-06-04 11:07:55,436 - DFI => Execução iniciada
2024-06-04 11:07:55,460 -     => Main - Execução iniciada
2024-06-04 11:07:55,461 -     => commodities - Execução iniciada
2024-06-04 11:08:23,790 - DFI => Execução iniciada
2024-06-04 11:08:23,818 -     => Main - Execução iniciada
2024-06-04 11:08:23,819 -     => commodities - Execução iniciada
2024-06-04 11:12:29,708 -     => commodities - Execução concluída
2024-06-04 11:12:29,722 -     => avacorp - Execução iniciada
2024-06-04 11:12:53,625 -     => avacorp - Execução concluída
2024-06-04 11:12:53,627 -     => Main - Execução concluída
2024-06-04 11:13:35,358 - DFI => Execução iniciada
2024-06-04 11:13:35,378 -     => Main - Execução iniciada
2024-06-04 11:13:35,378 -     => avacorp - Execução iniciada
2024-06-04 11:15:18,392 -     => avacorp - Execução concluída
2024-06-04 11:15:18,408 -     => Main - Execução concluída
2024-06-04 11:17:13,215 - DFI => Execução iniciada
2024-06-04 11:17:13,226 -     => Main - Execução iniciada
2024-06-04 11:17:13,227 -     => avacorp - Execução iniciada
2024-06-04 11:17:13,547 -       => Error:       (X) importDatabaseToDataframe.py.[executeImportDatabaseToDataframe]: Erro durante a importação de dados:: Index(...) must be called with a collection of some kind, '' was passed
2024-06-04 11:18:06,402 - DFI => Execução iniciada
2024-06-04 11:18:06,427 -     => Main - Execução iniciada
2024-06-04 11:18:06,428 -     => avacorp - Execução iniciada
2024-06-04 11:18:49,760 -       => Error:       (X) importDatabaseToDataframe.py.[executeImportDatabaseToDataframe]: Erro durante a importação de dados:: Index(...) must be called with a collection of some kind, '' was passed
2024-06-04 11:19:03,477 - DFI => Execução iniciada
2024-06-04 11:19:03,509 -     => Main - Execução iniciada
2024-06-04 11:19:03,510 -     => avacorp - Execução iniciada
2024-06-04 11:19:35,011 -       => Error:       (X) importDatabaseToDataframe.py.[executeImportDatabaseToDataframe]: Erro durante a importação de dados:: Index(...) must be called with a collection of some kind, '' was passed
2024-06-04 11:20:03,595 - DFI => Execução iniciada
2024-06-04 11:20:03,618 -     => Main - Execução iniciada
2024-06-04 11:20:03,619 -     => avacorp - Execução iniciada
2024-06-04 11:21:54,325 - DFI => Execução iniciada
2024-06-04 11:21:54,333 -     => Main - Execução iniciada
2024-06-04 11:21:54,333 -     => avacorp - Execução iniciada
2024-06-04 11:21:58,395 -     => avacorp - Execução concluída
2024-06-04 11:21:58,395 -     => Main - Execução concluída
2024-06-04 11:24:41,568 - DFI => Execução iniciada
2024-06-04 11:24:41,580 -     => Main - Execução iniciada
2024-06-04 11:24:41,580 -     => avacorp - Execução iniciada
2024-06-04 11:26:04,475 -     => avacorp - Execução concluída
2024-06-04 11:26:04,490 -     => Main - Execução concluída
2024-06-04 11:36:04,844 -     => Main - Execução iniciada
2024-06-04 11:36:04,845 -     => avacorp - Execução iniciada
2024-06-04 11:37:32,173 -     => avacorp - Execução concluída
2024-06-04 11:37:32,187 -     => Main - Execução concluída
2024-06-04 11:47:32,549 -     => Main - Execução iniciada
2024-06-04 11:47:32,549 -     => avacorp - Execução iniciada
2024-06-04 11:48:56,268 -     => avacorp - Execução concluída
2024-06-04 11:48:56,282 -     => Main - Execução concluída
2024-06-04 11:58:14,730 - DFI => Execução iniciada
2024-06-04 11:58:14,739 -     => Main - Execução iniciada
2024-06-04 11:58:14,739 -     => avacorp - Execução iniciada
2024-06-04 11:59:42,214 -     => avacorp - Execução concluída
2024-06-04 11:59:42,233 -     => Main - Execução concluída
2024-06-04 13:30:58,762 - DFI => Execução iniciada
2024-06-04 13:30:58,798 -     => Main - Execução iniciada
2024-06-04 13:30:58,799 -     => avacorp - Execução iniciada
2024-06-04 13:33:17,671 - DFI => Execução iniciada
2024-06-04 13:33:17,697 -     => Main - Execução iniciada
2024-06-04 13:33:17,698 -     => avacorp - Execução iniciada
2024-06-04 13:40:52,774 - DFI => Execução iniciada
2024-06-04 13:40:52,792 -     => Main - Execução iniciada
2024-06-04 13:40:52,793 -     => avacorp - Execução iniciada
2024-06-04 13:40:56,730 -     => avacorp - Execução concluída
2024-06-04 13:40:56,730 -     => Main - Execução concluída
2024-06-04 14:08:14,168 - DFI => Execução iniciada
2024-06-04 14:08:14,192 -     => Main - Execução iniciada
2024-06-04 14:08:14,192 -     => avacorp - Execução iniciada
2024-06-04 14:08:17,001 -       => Error:       (X) importDatabaseToDataframe.py.[executeImportDatabaseToDataframe]: Erro durante a importação de dados:: unsupported type for timedelta days component: str
2024-06-04 14:11:41,384 - DFI => Execução iniciada
2024-06-04 14:11:41,410 -     => Main - Execução iniciada
2024-06-04 14:11:41,411 -     => avacorp - Execução iniciada
2024-06-04 15:01:01,851 - DFI => Execução iniciada
2024-06-04 15:01:01,876 -     => Main - Execução iniciada
2024-06-04 15:01:01,877 -     => avacorp - Execução iniciada
2024-06-04 15:02:12,379 - DFI => Execução iniciada
2024-06-04 15:02:12,405 -     => Main - Execução iniciada
2024-06-04 15:02:12,405 -     => avacorp - Execução iniciada
2024-06-04 15:08:24,298 - DFI => Execução iniciada
2024-06-04 15:08:24,323 -     => Main - Execução iniciada
2024-06-04 15:08:24,324 -     => avacorp - Execução iniciada
2024-06-04 15:08:31,776 -     => avacorp - Execução concluída
2024-06-04 15:08:31,776 -     => Main - Execução concluída
2024-06-04 15:18:32,087 -     => Main - Execução iniciada
2024-06-04 15:18:32,089 -     => avacorp - Execução iniciada
2024-06-04 15:18:39,548 -     => avacorp - Execução concluída
2024-06-04 15:18:39,549 -     => Main - Execução concluída
2024-06-04 15:28:39,914 -     => Main - Execução iniciada
2024-06-04 15:28:39,916 -     => avacorp - Execução iniciada
2024-06-04 15:28:47,129 -     => avacorp - Execução concluída
2024-06-04 15:28:47,130 -     => Main - Execução concluída
2024-06-04 15:52:29,287 - DFI => Execução iniciada
2024-06-04 15:52:29,322 -     => Main - Execução iniciada
2024-06-04 15:52:29,324 -     => avacorp - Execução iniciada
2024-06-04 16:25:14,161 - DFI => Execução iniciada
2024-06-04 16:25:14,186 -     => Main - Execução iniciada
2024-06-04 16:25:14,187 -     => avacorp - Execução iniciada
2024-06-04 16:27:17,327 - DFI => Execução iniciada
2024-06-04 16:27:17,351 -     => Main - Execução iniciada
2024-06-04 16:27:17,352 -     => avacorp - Execução iniciada
2024-06-04 16:27:55,527 - DFI => Execução iniciada
2024-06-04 16:27:55,551 -     => Main - Execução iniciada
2024-06-04 16:27:55,552 -     => avacorp - Execução iniciada
2024-06-04 16:37:38,203 - DFI => Execução iniciada
2024-06-04 16:37:38,227 -     => Main - Execução iniciada
2024-06-04 16:37:38,228 -     => avacorp - Execução iniciada
2024-06-04 17:22:25,223 - DFI => Execução iniciada
2024-06-04 17:22:25,248 -     => Main - Execução iniciada
2024-06-04 17:22:25,249 -     => avacorp - Execução iniciada
2024-06-04 17:24:50,465 - DFI => Execução iniciada
2024-06-04 17:24:50,490 -     => Main - Execução iniciada
2024-06-04 17:24:50,491 -     => avacorp - Execução iniciada
2024-06-04 17:26:03,901 - DFI => Execução iniciada
2024-06-04 17:26:03,932 -     => Main - Execução iniciada
2024-06-04 17:26:03,933 -     => avacorp - Execução iniciada
